extensions/standard-processors/tests/integration/TailFileTest.cpp

```diff
@@ -69,9 +70,11 @@ class TailFileTestHarness : public IntegrationBase {
   }
 
   void runAssertions() override {
-    assert(LogTestController::getInstance().contains("5 flowfiles were received from TailFile input") == true);
-    assert(LogTestController::getInstance().contains("Looking for delimiter 0xA") == true);
-    assert(LogTestController::getInstance().contains("li\\ne5") == true);
+    using org::apache::nifi::minifi::utils::verifyLogLinePresenceInPollTime;
```

Comment:
I agree that having this function in our namespace is good, but in this case the `26` function level `using`s make me question whether we should drop a `using` at the top of the file, or go with the practice of having test utils at the top level

Reply:
I don't see any harm in having multiple using statements on demand, but if you like, I can move this into the integration testharness base class.

Reply:
I propose we leave this comment open a little and see if others have a preference, if nobody is for or against, we could leave it as is

Reply:
I counterpropose: This PR has been waiting for merge for quite a long time, we should finally merge it unless there isa major flaw to be found in it.

Reply:
I don't think it is rebased to `main`, keep it open until the CI finishes

Reply:
There is no conflict with main and all the jobs passed on Travis and Appveyor except from OSX 10.14.4. There is no compelling reason to have the GH Actions checks run them.

Reply:
feel free to mark this as resolved